src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -1394,19 +1373,39 @@ namespace SystemAvailabilityManager {
                     for (int extItem = 1; extItem <= numExtensibles; ++extItem) {
                         state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).AvailManagerName = "";
                         state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).cAvailManagerType = "";
-                        state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).AvailManagerType = 0;
+                        state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).AvailManagerType = DataPlant::SystemAvailabilityType::Invalid;
                     }
 
                     int listItem = 0;
-                    for (auto extensibleInstance : extensiblesArray) {
+                    for (const auto& extensibleInstance : extensiblesArray) {
                         ++listItem;
                         state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).AvailManagerName(listItem) =
                             ip->getAlphaFieldValue(extensibleInstance, extensionSchemaProps, "availability_manager_name");
                         std::string availManagerObjType =
                             ip->getAlphaFieldValue(extensibleInstance, extensionSchemaProps, "availability_manager_object_type");
                         state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).cAvailManagerType(listItem) = availManagerObjType;
-                        state.dataSystemAvailabilityManager->SysAvailMgrListData(Item).AvailManagerType(listItem) =
-                            ValidateAndSetSysAvailabilityManagerType(availManagerObjType);
+                        const auto& AvailManagerType = static_cast<DataPlant::SystemAvailabilityType>(
+                            getEnumerationValue(SystemAvailabilityTypeNamesUC, UtilityRoutines::MakeUPPERCase(availManagerObjType)));
+                        switch (AvailManagerType) {
```

Comment:
Delete this switch immediately. 😄

Reply:
This was a  weird attempt at accounting for this [if check](https://github.com/NREL/EnergyPlus/blob/6ec9b3bd683ea0535b823a75977dea39f4e91326/src/EnergyPlus/SystemAvailabilityManager.cc#L3893).  Changed to 
 ```
if (sysAvailMan.AvailManagerType(listItem) == DataPlant::SystemAvailabilityType::HybridVent)
                            sysAvailMan.AvailManagerType(listItem)  = DataPlant::SystemAvailabilityType::Invalid;
```
  instead.